samples/csharp/end-to-end-apps/StopSignDetection_ONNX/README.md

```diff
@@ -0,0 +1,165 @@
+2. Or, if you're using Visual Studio 2019, go to: _**Tools > Options > Environment > Preview Features**_ and check the box next to: _**Use previews of the .NET Core SDK**_
+
+## Code Walkthrough
+
+Create a class that defines the data schema to use while loading data into an `IDataView`. ML.NET supports the `Bitmap` type for images, so we'll specify `Bitmap` property decorated with the `ImageTypeAttribute` and pass in the height and width dimensions we got by [inspecting the model](#model-input-and-output), as shown below.
+
+```csharp
+public class StopSignInput
+{
+    public struct ImageSettings
+    {
+        public const int imageHeight = 320;
+        public const int imageWidth = 320;
+    }
+
+    public class StopSignInput
+    {
+        [ImageType(ImageSettings.imageHeight, ImageSettings.imageWidth)]
+        public Bitmap Image { get; set; }
+    }
+}
+```
+
+### ML.NET: Configure the model
+
+The first step is to create an empty `DataView` to obtain the schema of the data to use when configuring the model.
+
+```csharp
+var data = _mlContext.Data.LoadFromEnumerable(new List<StopSignInput>());
+```
+
+Next, we can use the input and output tensor names we got by [inspecting the model](#model-input-and-output) to define the **input** and **output** parameters of the ONNX Model. We can use this information to define the estimator pipeline. Usually, when dealing with deep neural networks, you must adapt the images to the format expected by the network. For this reason, the code below resizes and transforms the images (pixel values are normalized across all R,G,B channels). Since we have multiple outputs in our model, we can use the overload in **ApplyOnnxModel** to define a string array of output column names.
+
+```csharp
+var pipeline = context.Transforms.ResizeImages(resizing: ImageResizingEstimator.ResizingKind.Fill, outputColumnName: "image_tensor", imageWidth: ImageSettings.imageWidth, imageHeight: ImageSettings.imageHeight, inputColumnName: nameof(StopSignInput.Image))
+                .Append(context.Transforms.ExtractPixels(outputColumnName: "image_tensor"))
+                .Append(context.Transforms.ApplyOnnxModel(outputColumnNames: new string[] { "detected_boxes", "detected_scores", "detected_classes" }, 
+                    inputColumnNames: new string[] { "image_tensor" }, modelFile: "./Model/model.onnx"));
+```
+
+Last, create the model by fitting the `DataView`.
+
+```csharp
+var model = pipeline.Fit(data);
+```
+
+## Create a PredictionEngine
+
+After the model is configured, create a `PredictionEngine`, and then pass the image to the engine to classify images using the model.
+
+The **Console** app uses the `CreatePredictionEngine` to make predictions. Internally, it is optimized so the object dependencies are cached and shared across Http requests with minimum overhead when creating those objects.
```

Comment:
Did you mean PredictionEngine or PredictionEnginePool here? PredictionEngine shouldn't have anything to do with HTTP